### PR TITLE
Add auto width to .example-container for flex-shrink demo and media query for column layout

### LINKED ISSUE #1832: Fix width issue for .example-container demo output on flex-shrink page
Related to #1825 

https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink

### PR DESCRIPTION
**Pull request by tannerdolby**
Fixes #1832

### CODE REVIEW

**Comment by schalkneethling** [comment]
@tannerdolby Thanks so much for digging into this. I am wondering, is there really any benefit to that inner container? What if we made it like the example below:

https://user-images.githubusercontent.com/10350960/120669802-12856980-c490-11eb-845c-146764d40b3b.mov

**Comment by tannerdolby** [comment]
@schalkneethling Your welcome! I just came across it recently as felt we should provide a fix for it.

> I am wondering, is there really any benefit to that inner container? What if we made it like the example below:

To be honest, I don't think there is any benefit to that inner container. If we removed it, there would be much more available space for the demo output to occupy. Keeping the inner container and using `width: auto` like I suggested would work, but feels like a bandaid. Removing the inner container completely seems like the more sturdy fix.

**Comment by schalkneethling** [comment]
> Removing the inner container completely seems like the more sturdy fix.

Let’s do it. Thanks, @tannerdolby

**Comment by tannerdolby** [comment]
Ok deal. Your welcome @schalkneethling ! I will update my ~3 other PRs and start chugging away at removing inner containers for the `/flex/` interactive examples. 

Since almost all pages in `mdn/interactive-examples` need to have the inner container removed, would you prefer multiple PRs or one PR with many commits. Let me know! 👍

**Comment by schalkneethling** [comment]
> Since almost all pages in `mdn/interactive-examples` need to have the inner container removed, would you prefer multiple PRs or one PR with many commits?

Let’s do one with multiple commits.

File: live-examples/css-examples/flexbox/flex-shrink.css [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+        top: 35%;
+        transform: rotate(90deg);
+    }
+
+    #output {
+        height: auto;
+        width: auto;
+        margin-top: 10px;
+    }
+}
```
Comment (schalkneethling) [inline live-examples/css-examples/flexbox/flex-shrink.css:42]: As mentioned in another PR, I think this is a great idea but, let’s split it out of these changes and come to a consensus of how we can deal with this from a larger perspective and then make the changes based on our conclusion. Thanks!

**Review by schalkneethling — APPROVED** [review]
🚀

**Pull request merged**